.goreleaser.yml

```diff
@@ -0,0 +1,24 @@
+before:
+  hooks:
+    - make install
+builds:
```

Comment:
So we are basically writing here a different build script? Can we use our own script instead?